src/components/CreateWallet.jsx

```diff
@@ -64,47 +64,389 @@ const WalletCreationConfirmation = ({ createdWallet, walletConfirmed }) => {
+                  <rb.Form.Control.Feedback type="invalid">Word does not match.</rb.Form.Control.Feedback>
+                )}
+              </div>
+            </div>
+            <div className="row mb-3">
+              <div className="col">
+                <rb.InputGroup />
+                <rb.FormControl
+                  type="text"
+                  placeholder="Eleven"
+                  value={wordEleven}
+                  onChange={(e) => {
+                    setFeedback(false)
+                    setWordEleven(e.target.value)
+                  }}
+                  disabled={seedBackup}
+                  isInvalid={!wordEleven === seedphrase[10]}
+                  isValid={wordEleven === seedphrase[10]}
+                  required
+                />
+                {wordEleven === seedphrase[10] && !seedBackup ? (
+                  <rb.Form.Control.Feedback>Word matches!</rb.Form.Control.Feedback>
+                ) : (
+                  <rb.Form.Control.Feedback type="invalid">Word does not match.</rb.Form.Control.Feedback>
+                )}
+              </div>
+              <div className="col">
+                <rb.InputGroup />
+                <rb.FormControl
+                  type="text"
+                  placeholder="Twelve"
+                  value={wordTwelve}
+                  onChange={(e) => {
+                    setFeedback(false)
+                    setWordTwelve(e.target.value)
+                  }}
+                  disabled={seedBackup}
+                  isInvalid={!wordTwelve === seedphrase[11]}
+                  isValid={wordTwelve === seedphrase[11]}
+                  required
+                />
+                {wordTwelve === seedphrase[11] && !seedBackup ? (
+                  <rb.Form.Control.Feedback>Word matches!</rb.Form.Control.Feedback>
+                ) : (
+                  <rb.Form.Control.Feedback type="invalid">Word does not match.</rb.Form.Control.Feedback>
+                )}
+              </div>
+            </div>
+          </div>
+        </rb.Form>
+        {seedBackup ? <div className="text-center text-success">Seed phrase confirmed.</div> : <></>}
+        {feedback ? <div className="text-center text-danger">All words must match.</div> : <></>}
+        <div className="d-flex mt-4 mb-4 gap-3">
+          <rb.Button
+            variant="dark"
+            disabled={seedBackup}
+            onClick={() => {
+              setRevealSensitiveInfo(false)
+              setSensitiveInfoWasRevealed(false)
+              setUserConfirmed(false)
+              setStep(0)
+            }}
+          >
+            Back
+          </rb.Button>
+          {seedBackup === false ? (
+            <rb.Button variant="dark" onClick={() => onSubmit()}>
+              Verify
+            </rb.Button>
+          ) : (
+            <rb.Button variant="dark" onClick={() => userConfirmed && walletConfirmed()}>
+              Fund wallet
+            </rb.Button>
+          )}
```

Comment:
I feel like we could probably go without the extra "verify" step and just show the "fund" button (disabled as long as the entered seedphrase doesn't match, of course).

Reply:
Good idea, I've made it smoother and more reactive now.